.github/PULL_REQUEST_TEMPLATE.md

```diff
@@ -0,0 +1,10 @@
+**Which issue is resolved by this Pull Request:** 
+Resolves #
+
+**Description of your changes:**
+
+**Environment:**
```

Comment:
maybe call this `"Environments tested"`?

Reply:
thanks